### ISSUE TITLE
Social Media Manager ernennen

### ISSUE THREAD

**Issue by hmitsch** [MozillaDE/ziele2016#3]
Klärung, wer die "mindestens 3 Personen" sind, welche sich an der Unterhaltung der Social Media Konten beteiligen.

Konten:
- Twitter: @mozilla_deutsch
- Twitter: @MozillaDe
- Twitter: @FirefoxDe
- Facebook: mozilla.de-Seite

**Comment by hmitsch** [comment]
Ich könnte mich für @MozillaDe melden. Kann mir vorstellen, da etwa 1 Tweet pro Woche rauszulassen.

**Comment by cadeyrn** [comment]
Mir macht die Struktur mit den drei Accounts noch Sorgen. "mozilla_deutsch" und "MozillaDe" klingt vom Namen her, als würde beides das Gleiche abdecken. Und die Frage wäre außerdem, was bietet man über diese Accounts an, wenn es auch noch FirefoxDe gibt? Alles, was FirefoxDe auch twittert plus mehr oder alles, was FirefoxDe nicht twittert oder gibt es Überschneidungen und wenn ja, wo sind die? Vielleicht sollten wir erst noch die Rolle der einzelnen Accounts klären. :)

**Comment by MichaelKohler** [comment]
Ja, dafür würde ich einen separaten Issue machen. Da braucht es definitiv ein Konzept aus meiner Sicht. Magst du?

**Comment by hmitsch** [comment]
Issue angelegt: https://github.com/MozillaDE/ziele2016/issues/4

**Comment by hmitsch** [comment]
@cadeyrn: Super Kommentar. Sehe ich genau so wie du!
